daemon/graphdriver/vfs/driver.go

```diff
@@ -75,6 +99,16 @@ func (d *Driver) Remove(id string) error {
 
 func (d *Driver) Get(id, mountLabel string) (string, error) {
 	dir := d.dir(id)
+	if strings.Contains(id, "-init") {
+		idInitFile := d.initFile(id)
+		if _, err := os.Stat(idInitFile); err == nil {
+			data, err := ioutil.ReadFile(idInitFile)
+			if err != nil {
+				return "", err
+			}
+			dir = string(data)
+		}
+	}
```

Comment:
Sorry, I think I understand the change in `Create`, but not that one in `Get`. Are we effectively making it impossile to `Get` the init layer? Is that desirable?